src/client/Client.js

```diff
@@ -195,7 +195,9 @@ class Client extends BaseClient {
 
   /**
    * Logs the client in, establishing a websocket connection to Discord.
-   * @param {string} token Token of the account to log in with
+   * <info>If no token is provided and there hasn't been a connection yet, `process.env.DISCORD_TOKEN`
+   * will be used.</info>
+   * @param {string} [token] Token of the account to log in with
```

Comment:
```suggestion
   * @param {string} [token=this.token] Token of the account to log in with
```
i believe this should be added